src/appcast.cpp

```diff
@@ -256,7 +276,7 @@ void XMLCALL OnText(void *data, const char *s, int len)
     const size_t size = ctxt.items.size();
 
     if ( ctxt.in_relnotes )
-        ctxt.items[size-1].ReleaseNotesURL.append(s, len);
+        ctxt.items[size-1].ReleaseNotesURL.assign(s, len);
```

Comment:
I understand why you did this, but you need to avoid concatenating different links in another way. This is wrong, because Expat makes no guarantees about how often does it call this handler. It may decide to split any text block in multiple pieces, hence `append`. For example, it would be reasonable to clear `ReleaseNotesURL` when the tag is opened anew.